io/serialport/25-serial.js

```diff
@@ -41,6 +41,20 @@ module.exports = function(RED) {
             node.port = serialPool.get(this.serialConfig);
 
             node.on("input",function(msg) {
+				if (msg.hasOwnProperty("baudrate")) {
+					var baud = parseInt(msg.baudrate);
+					if (isNaN(baud)) {
+						var errmsg = err.toString().replace("Serialport","Serialport "+node.port.serial.path);
+						node.error(errmsg,"Cannot parse baudrate property");
```

Comment:
the errmsg should be the text you have, and the second parameter should just be the incoming msg.
They should also be translatable - the text should be in the locale file - eg see line 61 etc.

Reply:
Changed to be like line 63